docs/tutorial/using-checkbox/test-report.rst

```diff
@@ -0,0 +1,87 @@
+.. _test-report:
+
+Review Test Report 
+======================
```

Comment:
Minor: make complains, too many equals

Reply:
fixed in [fdd5e20](https://github.com/canonical/checkbox/pull/637/commits/fdd5e208e18dfabfacf2c8bf416d1b017b73251b)